packaging/rpm/microshift.spec

```diff
@@ -127,6 +126,23 @@ Requires: greenboot
 %description greenboot
 The microshift-greenboot package provides the Greenboot scripts used for verifying that MicroShift is up and running.
 
+%package microshift-olm-release-info
+Summary: Release information for MicroShift operator-lifecycle-manager
+BuildArch: noarch
+Requires: microshift = %{version}
+
+%description microshift-olm-release-info
+The microshift-olm-release-info package provides release information files for this
+release. These files contain the list of container image references used by
+the Operator-lifecycle-manager and can be used to embed those images into osbuilder blueprints.
+
+%package microshift-olm
+Summary: Operator-lifecycle-manager manifests for microshift
+BuildArch: noarch
```

Comment:
I'm wondering if we need two packages. Could `microshift-olm` contain the `release-$ARCH.json` and we'd have just one? It only contains text files so I think we could get away with one RPM

Reply:
Sounds good to me!